dockerfile/Dockerfile

```diff
@@ -13,62 +13,59 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License
-
+#
 # ------------------------------------------------------------------------
 
+# set to latest Ubuntu LTS
 FROM ubuntu:16.04
 MAINTAINER WSO2 Docker Maintainers "dev@wso2.org"
-ENV DEBIAN_FRONTEND noninteractive
-
-# defined arguments and defaults
-ARG USER=wso2user
 
+# set user configurations
+ARG USER=wso2carbon
+ARG USER_GROUP=wso2
+ARG USER_HOME=/home/${USER}
+# set dependant files directory
 ARG FILES=./files
-ARG USER_HOME=/home/wso2user
+# set jdk configurations
+ARG JDK_ARCHIVE=jdk-8u*-linux-x64.tar.gz
+ARG JAVA_HOME=${USER_HOME}/java
+# set wso2 product configurations
 ARG WSO2_SERVER=wso2das
 ARG WSO2_SERVER_VERSION=3.1.0
 ARG WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}.zip
-ARG JDK_ARCHIVE=jdk-8u*-linux-x64.tar.gz
-ARG JAVA_HOME=${USER_HOME}/java
+ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}-${WSO2_SERVER_VERSION}
 
-RUN apt-get update \
-    	&& apt-get install -y --no-install-recommends --no-install-suggests \
-       	zip \
-        unzip \
-        telnet \
-        iproute2 \
-    	sudo && rm -rf /var/lib/apt/lists/*
+# install required packages
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends --no-install-suggests unzip curl telnet iproute2 && \
```

Comment:
Shall we move the packages to one line to make it more readable? WDYT ?

Reply:
@DilanUA I think we should stick to having packages in multiple lines. This is primarily because this seems to be the practice adopted in Docker library Dockerfiles. Please see the following Dockerfile best practices guide example [1] and openJDK official Dockerfile [2], respectively.
[1]: [link](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#sort-multi-line-arguments)
[2]: [link](https://github.com/docker-library/openjdk/blob/a893fe3cd82757e7bccc0948c88bfee09bd916c3/8-jdk/Dockerfile#L19)
Plus, IMO this technique is better since no other package names are expected to be added.